components/script/dom/webglrenderingcontext.rs

```diff
@@ -513,19 +547,20 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
     #[allow(unsafe_code)]
     // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.3
     fn GetParameter(&self, cx: *mut JSContext, parameter: u32) -> JSVal {
-        // Handle the GL_FRAMEBUFFER_BINDING without going all the way
+        // Handle the GL_*_BINDING without going all the way
         // to the GL, since we would just need to map back from GL's
-        // returned ID to the WebGLFramebuffer we're tracking.
+        // returned ID to the WebGL* object we're tracking.
         match parameter {
-            constants::FRAMEBUFFER_BINDING => {
-                rooted!(in(cx) let mut rval = NullValue());
-                if let Some(bound_fb) = self.bound_framebuffer.get() {
-                    unsafe {
-                        bound_fb.to_jsval(cx, rval.handle_mut());
-                    }
-                }
-                return rval.get()
+            constants::ARRAY_BUFFER_BINDING => return object_binding_to_js_or_null!(cx, &self.bound_buffer_array),
+            constants::ELEMENT_ARRAY_BUFFER_BINDING => {
+                return object_binding_to_js_or_null!(cx, &self.bound_buffer_element_array);
             }
+            constants::FRAMEBUFFER_BINDING => return object_binding_to_js_or_null!(cx, &self.bound_framebuffer),
+            constants::RENDERBUFFER_BINDING => return object_binding_to_js_or_null!(cx, &self.bound_renderbuffer),
+            constants::TEXTURE_BINDING_2D => return object_binding_to_js_or_null!(cx, &self.bound_texture_2d),
+            constants::TEXTURE_BINDING_CUBE_MAP => {
+                return object_binding_to_js_or_null!(cx, &self.bound_texture_cube_map);
```

Comment:
If the braces are just due to a line length issue, why not formatting it like this to be consistent?

```rust
constants::ARRAY_BUFFER_BINDING
    => return object_binding_to_js_or_null!(cx, &self.bound_buffer_array),
constants::ELEMENT_ARRAY_BUFFER_BINDING
    => return object_binding_to_js_or_null!(cx, &self.bound_buffer_element_array),
constants::FRAMEBUFFER_BINDING
    => return object_binding_to_js_or_null!(cx, &self.bound_framebuffer),
// ...
```